src/store/modules/nft.js

```diff
@@ -494,13 +494,18 @@ const actions = {
       uri,
       parent,
       metadata: classMetadata = {},
+      owner,
     } = classData;
     const formattedMetadata = {
       name,
       description,
       uri,
       ...classMetadata,
       parent,
+      // HACK: owner field only exists in /likechain/likenft/v1/class endpoint,
+      // not in /cosmos/nft/v1beta1/classes/:classId endpoint.
+      // Should be removed after ISCN data refactor
+      owner,
```

Comment:
Should we rename this to iscn_owner here to reduce code change?

Reply:
Seems linter doesn't allow me to use camel case.

Reply:
add a eslint ignore next line for this case